### PR TITLE
DSND-3100: Update structured logging for service

### PR DESCRIPTION
**Pull request by kinpang-CH**
* Incorporate request id into logMap
* Update wording on officers search logging
* Update pom.xml to use different build command

Resolves [DSND-3100](https://companieshouse.atlassian.net/browse/DSND-3100)

[DSND-3100]: https://companieshouse.atlassian.net/browse/DSND-3100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: pom.xml [hunk 1]
```diff
@@ -246,6 +246,19 @@
                 <artifactId>sonar-maven-plugin</artifactId>
                 <version>${sonar-maven-plugin.version}</version>
             </plugin>
+            <plugin>
```
Comment (cgregoryCH) [inline pom.xml:249]: This is a duplicate dependency off line 223. The one above is just missing the config.

Comment (kinpang-CH) [reply pom.xml:244]: Resolved in latest commit

**Review by cgregoryCH — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sthompsonCH — APPROVED** [review]

**Pull request merged**